tests/functional/test_logger.py

```diff
@@ -251,6 +251,21 @@ def test_logger_append_duplicated(stdout, service_name):
     assert "new_value" == log["request_id"]
 
 
+def test_logger_honors_given_exception_keys(stdout, service_name):
+    # GIVEN Logger is initialized with exception and exception_name fields
+    logger = Logger(
+        service=service_name, stream=stdout, exception="exception_value", exception_name="exception_name_value"
+    )
+
+    # WHEN log level info
+    logger.info("log")
```

Comment:
Suggestion: add the exception fields here so it's easier to scan and understand what's being tested.

logger.info("log", exception=..., exception_name=...)

Reply:
Good catch @heitorlessa! Thank you!